packages/angular-imask/src/imask.directive.ts

```diff
@@ -190,6 +192,10 @@ export class IMaskDirective<Opts extends IMask.AnyMaskedOptions> implements Cont
   }
 
   private _isAndroid(): boolean {
+    // secure access to navigator on server renders
+    if (isPlatformServer(this._platformId)) {
+      return false;
+    }
     return /android (\d+)/.test(navigator.userAgent.toLowerCase());
```

Comment:
You can one-line this as following:
```ts
return isPlatrformBrowser(this._platformId) && /android (\d+)/.test(navigator.userAgent.toLowerCase());
```

Reply:
yes, done.

https://github.com/uNmAnNeR/imaskjs/pull/570/commits/35b9069832270bcdca0671436754848b53e55202#diff-5a0dc28acf4f1e69d882fe8fc38c3c55ac1d21a2a5b3a18144f498ac8926f6b7R195

Reply:
It's either `!isPlatformServer` or `isPlafromBrowser` :)

Reply:
thanks, read too fast